docs/intelligentapps/evaluation.md

```diff
@@ -0,0 +1,36 @@
+---
+Order: 5
+Area: intelligentapps
+TOCTitle: Evaluation
+ContentId:
+PageTitle: AI Evaluation
+DateApproved:
+MetaDescription: Import a dataset with LLMs or SLMs output or rerun it for the queries. Run evaluation job for the popular evaluators like F1 score, relevance, coherence, similarity... find, visualize, and compare the evaluation results in tables or charts.
+MetaSocialImage:
+---
+
+# Evaluation Overview
+
+AI engineers often need to evaluate models with different parameters, different prompts in a dataset for comparing to ground truth and generate values from the comparisons. AI Toolkit allows you do perform evaluations with minimal efforts.
+
+## Prepare an evaluation job
+
+Navigate to the tree view's `Batch Run` entry in `TOOLS` section. You will see an evaluation job history view. Select the button `Start Evaluation`, then answer the questions:
```

Comment:
Use numbered steps to provide instructions, don't just add a sentence because it's not very easy to parse.
Include a screenshot where needed